packages/grafana-data/src/themes/createShadows.ts

```diff
@@ -8,9 +8,9 @@ export interface ThemeShadows {
 }
 
 function createDarkShadow(...px: number[]) {
-  const shadowKeyUmbraOpacity = 0.2;
-  const shadowKeyPenumbraOpacity = 0.14;
-  const shadowAmbientShadowOpacity = 0.12;
+  const shadowKeyUmbraOpacity = 0.5;
+  const shadowKeyPenumbraOpacity = 0.4;
+  const shadowAmbientShadowOpacity = 0.3;
```

Comment:
These names need a Wikipedia reference in a comment ;) https://en.wikipedia.org/wiki/Umbra,_penumbra_and_antumbra